ModTek/Patches/ActivateAfterInit.cs

```diff
@@ -0,0 +1,32 @@
+using Harmony;
+using UnityEngine;
+
+namespace ModTek
+{
+    /// <summary>
+    /// Disable activateAfterInit from functioning for the Start() on the "Main" game object which activates the BattleTechGame object
+    /// This stops the main game object from loading immediately -- so work can be done beforehand
+    /// </summary>
+    [HarmonyPatch(typeof(ActivateAfterInit), "Start")]
+    public static class ActivateAfterInit_Start_Patch
```

Comment:
I would call the file names of these patches the same as the class names inside. ActivateAfterInit_Start_Patch.cs

Reply:
I rather like them being named after the classes that they are patching, since they could have multiple patches in each file.

Reply:
Fair enough